element_calcium_imaging/plotting/widget.py

```diff
@@ -0,0 +1,205 @@
+        },
+        xaxis={
+            "title": "X (px)",
+            "visible": True,
+            "showticklabels": True,
+            "showgrid": False,
+        },
+        yaxis={
+            "title": "Y (px)",
+            "visible": True,
+            "showticklabels": True,
+            "showgrid": False,
+        },
+        paper_bgcolor="rgba(0,0,0,0)",
+        plot_bgcolor="rgba(0,0,0,0)",
+        modebar_remove=[
+            "zoom",
+            "resetScale",
+            "pan",
+            "select",
+            "zoomIn",
+            "zoomOut",
+            "autoScale2d",
+        ],
+        shapes=[
+            go.layout.Shape(
+                type="rect",
+                xref="paper",
+                yref="paper",
+                x0=0.035,
+                y0=0,
+                x1=0.965,
+                y1=1.0,
+                line={"width": 1, "color": "black"},
+            )
+        ],
+    )
+    fig1 = go.Figure(
+        go.Image(
+            z=None,
+            hovertemplate="x: %{x} <br>y: %{y} <br>mask_id: %{customdata} <extra></extra>",
+            customdata=None,
+        ),
+        layout=FIG1_LAYOUT,
+    )
+
+    FIG2_WIDTH = 600
+    FIG2_HEIGHT = 600
+    fig2_layout = cell_plot.get_tracelayout(None, width=FIG2_WIDTH, height=FIG2_HEIGHT)
+
+
+    fig2 = go.Figure(
+        [
+            go.Scatter(
+                x=None,
+                y=None,
+                name="Fluorescence",
+                yaxis="y1",
+            ),
+            go.Scatter(x=None, y=None, name="Calcium Event", yaxis="y2"),
+        ],
+        layout=fig2_layout,
+    )
+
+    fig1_widget = go.FigureWidget(fig1)
+    fig2_widget = go.FigureWidget(fig2)
+
+    def tooltip_click(trace, points, selector):
+        mask_id = trace.customdata[points.ys[0]][points.xs[0]]
+
+        if mask_id > -1:
+            activity_trace_figobj = from_json(
+                (
+                    ActivityReport
+                    & motioncorrection_dropdown.value
+                    & f"mask='{mask_id}'"
+                ).fetch1("activity_trace")
+            )
+
+            with fig2_widget.batch_update():
+                fig2_widget.data[0].x = activity_trace_figobj.data[0].x
+                fig2_widget.data[0].y = activity_trace_figobj.data[0].y
+                fig2_widget.data[0].name = activity_trace_figobj.data[0].name
+                fig2_widget.data[1].x = activity_trace_figobj.data[1].x
+                fig2_widget.data[1].y = activity_trace_figobj.data[1].y
+                fig2_widget.data[1].name = activity_trace_figobj.data[1].name
+                fig2_widget.layout["title"] = {
+                    "text": f"Trace for Cell {mask_id}",
+                    "xanchor": "center",
+                    "yanchor": "top",
+                    "y": 0.97,
+                    "x": 0.5,
+                }
+
+    def response(change, usedb=False):
+        if usedb:
+            composite_average_image = from_json(

```

Comment:
```suggestion
            cell_overlayed_image = from_json(
```

Reply:
Minor name change to keep the variable names consistent for improved readability.